api/lib/block-listeners/cosmos-node-subscription.js

```diff
@@ -17,6 +17,7 @@ const EXPECTED_MAX_BLOCK_WINDOW = 120000
 // if we don't do this, we run into errors as the data is not yet available
 const COSMOS_DB_DELAY = 2000
 const PROPOSAL_POLLING_INTERVAL = 600000 // 10min
+const UPDATE_VALIDATORS_INTERVAL = 600000 // 10min
```

Comment:
so validators would only be visible after 10 minutes?

Reply:
no, because it runs it just on start. But they update every 10 minutes yes. I mean, like nearly everything I push I tested it and saw it works